packages/overlay/src/ActiveOverlay.ts

```diff
@@ -426,6 +426,16 @@ export class ActiveOverlay extends SpectrumElement {
         this.setOverlayPosition();
     };
 
+    public resetOverlayPosition = (): void => {
+        this.style.setProperty('max-height', '');
```

Comment:
Are `style.setProperty('max-height', '')` and `style.removeProperty('max-height')` equivalent?

Reply:
Oooh, I've not used that API, it seems like it should be, so I'll test that as it seems like a nicer future to maintain.